index.js

```diff
@@ -1,7 +1,6 @@
 const ObservableStore = require('obs-store');
 const equal = require('fast-deep-equal');
-const clone = require('clone');
-const generateRandomString = require('crypto-random-string');
+const uuid = require('uuid/v4');
```

Comment:
After I started using `uuid` for the permission object IDs, I noticed you used `crypto-random-string` for metadata. I decided to go for [`uuid`](https://www.npmjs.com/package/uuid) because it's more popular and implements a standard. Both are common heuristics for me, but only heuristics.